packages/prices/package.json

```diff
@@ -0,0 +1,33 @@
+{
```

Comment:
Let's move this into `examples/xnft-program-library`, which should turn into the set of all pre-installed xNFTs.